bucket/vim-nightly.json

```diff
@@ -3,6 +3,7 @@
     "description": "A highly configurable text editor for efficiently creating and changing any kind of text.",
     "license": "Vim",
     "version": "8.2.0324",
+    "notes": "Add gVim as a context menu option by running: $dir\\install-context.reg",
```

Comment:
```suggestion
    "notes": "Add gVim as a context menu option by running: '$dir\\install-context.reg'",
```

Reply:
I didn't add quotes because it makes it harder to copy paste -- double clicking selects the quotes.
Is that acceptable or do you still want me to change?

Reply:
Not all terminals linkify paths without quotes

Reply:
Double quotes still copy-paste correctly, so I used them. (Single quotes produce an error when copy pasted in cmd.exe).